azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -656,7 +713,7 @@ def _add_to_actions(self, action_repr: Union[List[Action], Action]):
 
         if self._replay_schema is ReplaySchema.V1 and isinstance(action_repr, list):
             self._action_payload_v1.append(action_repr)
-        elif self._replay_schema is ReplaySchema.V2 and isinstance(action_repr, Action):
+        elif self._replay_schema.value >= ReplaySchema.V2.value and isinstance(action_repr, Action):
```

Comment:
nit - is accessing `.value` necessary? The `if-` statement right above uses `self.replay_schema` is `ReplaySchema.V1`

Reply:
"is" operator works, comparison (>, <, etc) do not. Not sure why

Reply:
ah yes, this makes sense, thanks!